Source/.clang-format

```diff
@@ -66,8 +66,8 @@ PenaltyBreakString: 1000
 PenaltyExcessCharacter: 1000000
 PenaltyReturnTypeOnItsOwnLine: 60
 PointerAlignment: Left
-ReflowComments:  true
-SortIncludes:    true
+ReflowComments:  Always
+SortIncludes:    CaseInsensitive
```

Comment:
Our current behavior seems to be `CaseSensitive`, but I don't know if we have any situations where this will actually make a difference.